README.md

```diff
@@ -63,6 +73,12 @@ var frLanguage = {
       }
       return tokenScore;
     }
+  },
+  getDictionary: {
+    apply: function() {
+      // Load a dictionary for the language for nspell to use
+      return { aff, dic };
```

Comment:
What do these `aff` and `dic` properties correspond to? Could you give an example?

Reply:
Explained more in the README file

Reply:
This doesn't seem to be the case, are you sure that you have pushed your changes?

Reply:
new commit, it references the explanation I gave in the "Spell checked example" section